### PR TITLE
Update JS rules and linter

### PR DESCRIPTION
**Pull request by gabrielpoca**
I believe that most of us agree that airbnb's eslint configuration
should be our default linter for JavaScript. I know that, for some
projects, we may feel that tweaking it would be better, but I believe
that we should agree in a set of rules and follow them. We'll never find
something that everyone likes at first. I also think that we need a
default configuration because it will remove this conversation out of
the way, and let us focus on different things. I also think that we need
consistency in our projects, and we need to remove the amount of
decisions we make whenever we start a new one. This doesn't mean that
this topic is closed forever, we should be open to new ideas, but keep
in mind that the benefits of using one styleguide or the other are
probably very small. Most times, I believe, it's a matter of taste.

### CODE REVIEW

File: style/js/README.md [hunk 1]
```diff
@@ -161,10 +161,4 @@ if (!foo) {
...
 ------------
 
-* Initialize arrays using `[]`.
-* Initialize empty objects and hashes using `{}`.
-* Use `CamelCase` for classes, `lowerCamelCase` for variables and functions,
-  `SCREAMING_SNAKE_CASE` for constants, `_single_leading_underscore` for
-  private variables and functions.
-* Prefer `== ` or `===` to `is`.
-* Prefer `||` and `&&` to `or` and `and`.
+* Don't use CoffeScript
```
Comment (naps62) [inline style/js/README.md]: how about removing this section altogether?

**Review by naps62 — APPROVED** [review]

File: style/js/README.md (hunk 1 above)
Comment (gabrielpoca) [reply style/js/README.md]: 👍

File: linters/js/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ESLint Setup
+============
+
+1. Copy [.eslintrc](.eslintrc) to the root of the project.
+2. Install `eslint-config-airbnb`.
+
+
+    npm i --save-dev eslint-config-airbnb
```
Comment (frm) [inline linters/js/README.md:8]: Should we support yarn or add yarn docs?

Comment (gabrielpoca) [reply linters/js/README.md:8]: I think that conversations needs to happen, but not here. At the moment, some people are experimenting with `yarn`, but our default remains `npm`.

Comment (frm) [reply linters/js/README.md:8]: 👍 I'm fine with that.

**Review by frm — APPROVED** [review]

**Comment by joaojusto** [comment]
Airbnb seems a very good starting point :shipit:

**Comment by gabrielpoca** [comment]
@subvisual/developers last call for comments 🍾

**Comment by gabrielpoca** [comment]
After discussing with Azevedo, I changed the configuration to override two of Airbnb's rules. The rules don't affect the styling; it just makes it a little less strict. You'll find an explanation in the README. /cc @subvisual/developers

File: linters/js/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+
+3. Install its peer dependencies following the [instructions in the repo](https://www.npmjs.com/package/eslint-config-airbnb)
+
+
+Overrides
+---------
+
+Airbnb's configuration is great, but some of their rules are to strict for our
```
Comment (azevedo) [inline linters/js/README.md]: "... `too` strict ..."

**Review by azevedo — CHANGES_REQUESTED** [review]

File: linters/js/README.md (hunk 3 above)
Comment (gabrielpoca) [reply linters/js/README.md]: 👍

**Review by azevedo — APPROVED** [review]

**Review by naps62 — APPROVED** [review]

**Review by pfac — APPROVED** [review]

**Review by ronaldofs — APPROVED** [review]

**Pull request merged**